cmds/core/yes/yes_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"bufio"
+	"testing"
+
+	"github.com/u-root/u-root/pkg/testutil"
+)
+
+func TestMain(m *testing.M) {
+	testutil.Run(m, main)
+}
+
+func TestYes(t *testing.T) {
+	cmd := testutil.Command(t)
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		t.Fatalf("cmd.StdoutPipe: got %v, want nil", err)
+	}
+
+	if err := cmd.Start(); err != nil {
+		t.Fatalf("cmd.Start: got %v, want nil", err)
+	}
+	in := bufio.NewScanner(stdout)
+	for i := 0; i < 1000; i++ {
+		if !in.Scan() {
+			t.Fatalf("Could not scan: got %v, want nil", in.Err())
+		}
+		if text := in.Text(); text != "y" {
+			t.Errorf("Got %v, want 'y'", text)
```

Comment:
nit: might be useful to print the iteration where it failed.

Reply:
good idea.

Reply:
That is a very good idea